packages/spor-react/src/theme/slot-recipes/tabs.ts

```diff
@@ -46,6 +75,15 @@ export const tabsSlotRecipe = defineSlotRecipe({
           },
```

Comment:
Hover får heller ikke helt riktige verdier. Prøv dette isteden:
```suggestion
          _hover: {
            outline: "2px solid",
            outlineColor: "core.surface.hover",
            outlineOffset: "-2px",
          },
```